.circleci/config.yml

```diff
@@ -186,7 +186,7 @@ jobs:
           name: Run integration tests
           command: |
             cp .env.integration-tests .env
-            make refresh SKIP_DUMMY=1 PYTEST_SENTRY_DSN=$PYTEST_SENTRY_DSN up_prod_detached integration_test_nimbus PYTEST_ARGS="$PYTEST_ARGS"
+            make refresh SKIP_DUMMY=1 PYTEST_SENTRY_DSN=$PYTEST_SENTRY_DSN CIRCLECI=$CIRCLECI up_prod_detached integration_test_nimbus PYTEST_ARGS="$PYTEST_ARGS"
```

Comment:
I understand the reason of adding`PYTEST_ARGS` but I am not about the reason behind adding `CIRCLECI`

Reply:
Yes, this was added to pass in to the ```selenium``` fixture, so that we can skip the experiment clean up step on CI